src/components/Card.js

```diff
@@ -94,7 +94,8 @@ const Card = ({ content }) => {
           </Box>
           {subItems}
         </Box>
-        <Box>{downLoadButton}</Box>
+        {/* temporarily Hide download data buttons */}
```

Comment:
Suggest that going forward, you reference the issue if you're temp-hiding things.